app/javascript/components/search/controls/SearchPanel.js

```diff
@@ -109,69 +50,31 @@ export default function SearchPanel({
   })
 
   /** helper method as, for unknown reasons, clicking the bootstrap modal auto-scrolls the page down */
-  function closeModal(modalShowFunc) {
-    modalShowFunc(false)
+  function closeSearchModal() {
+    setShowSearchHelpModal(false)
     setTimeout(() => {scrollTo(0, 0)}, 0)
   }
 
-  /** handle enabling/disabling advanced search on the page and syncing the user's feature flags */
-  function setAdvancedSearchEnabled(enabled, modalShowFunc) {
-    // for signed-in users, update their feature flag, for everyone, save in localStorage
-    // this means signed-in users will see advanced search even if they are not signed in, once they
-    // opt in
-
-    if (!userState.isAnonymous) {
-      userState.updateFeatureFlags({ faceted_search: enabled })
-    }
-    localStorage.setItem('faceted-search-flag', enabled.toString())
-    setShowAdvancedSearch(enabled)
-    setIsNewToUser(false)
-    if (modalShowFunc) {
-      closeModal(modalShowFunc)
-    }
-    if (enabled) {
-      selectionContext.updateSelection({ terms: '' }, true)
-    }
-  }
-
   return (
     <div id='search-panel'>
       <KeywordSearch keywordPrompt={keywordPrompt}/>
-      { searchButtons }
-      { advancedOptsLink }
-      { downloadButtons }
-      <Modal
-        show={showSearchOptInModal}
-        onHide={() => closeModal(setShowSearchOptInModal)}
-        animation={false}
-        bsSize='large'>
-        <Modal.Body className="">
-          { helpModalContent }
-        </Modal.Body>
-        <Modal.Footer>
-          <button className="btn btn-md btn-primary" onClick={() => {
-            setAdvancedSearchEnabled(true, setShowSearchOptInModal)
-          }}>Yes, show advanced search</button>
-          <button className="btn btn-md" onClick={() => {
-            setAdvancedSearchEnabled(false, setShowSearchOptInModal)
-          }}>Cancel</button>
-        </Modal.Footer>
-      </Modal>
+      <FacetsPanel/>
+      <a className="action advanced-opts"
+        onClick={() => setShowSearchHelpModal(true)}
+        data-analytics-name="search-help">
+        <FontAwesomeIcon icon={faQuestionCircle} />
+      </a>
+      <DownloadProvider><DownloadButton /></DownloadProvider>
       <Modal
         show={showSearchHelpModal}
-        onHide={() => closeModal(setShowSearchHelpModal)}
+        onHide={closeSearchModal}
         animation={false}
         bsSize='large'>
         <Modal.Body className="">
           { helpModalContent }
         </Modal.Body>
         <Modal.Footer>
-          <button className="btn btn-md btn-primary" onClick={() => {
-            setAdvancedSearchEnabled(true, setShowSearchHelpModal)
-          }}>Yes, use advanced search</button>
-          <button className="btn btn-md" onClick={() => {
-            setAdvancedSearchEnabled(false, setShowSearchHelpModal)
-          }}>Go back to legacy search</button>
+          <button className="btn btn-md btn-primary" onClick={closeSearchModal}>Ok</button>
```

Comment:
Maybe of interest:

The SCP UI uses both "OK" and "Ok".  (Same with [Terra](https://github.com/DataBiosphere/terra-ui/search?q=OK).)  Chrome, macOS, and [Unicode](https://www.compart.com/en/unicode/U+1F197) all use "OK", but "Ok" is also commonly seen in various products.  I'd opt for "OK", but would be fine with "Ok", as long as we're consistent.